indexer/data_source.hpp

```diff
@@ -89,6 +89,11 @@ class FrozenDataSource : public DataSource
 
 /// Guard for loading features from particular MWM by demand.
 /// @note This guard is suitable when mwm is loaded.
+/// @note If you need to work with |FeatureType| from different threads you need to use
```

Comment:
nit: I don't think we need the || brackets for types, they are easy to read as is. The brackets help for lowercase value indentifiers.

Reply:
Ok. I removed the brackets you're talking about.